### PR TITLE
Module6

### PR DESCRIPTION
**Pull request by lbrucher**

### CODE REVIEW

File: exercises/module6/6.2/.gitignore [hunk 1]
```diff
@@ -0,0 +1,12 @@
+node_modules/
+coverage/
+
+.DS_Store
+.env.local
```
Comment (e-baron) [inline exercises/module6/6.2/.gitignore]: .env to be added else our student's secrets will be on github.

File: exercises/module6/6.2/requests.http [hunk 2]
```diff
@@ -0,0 +1,75 @@
+@hostname = localhost
+@port = 3001
+@host = http://{{hostname}}:{{port}}
+@personId = 3
```
Comment (e-baron) [inline exercises/module6/6.2/requests.http]: The API works with UUID, not with integer. Therefore this script might be misleading...

File: exercises/module6/6.2/.gitignore (hunk 1 above)
Comment (lbrucher) [reply exercises/module6/6.2/.gitignore]: updated

**Review by e-baron — APPROVED** [review]
LGTM, thanks : )

**Pull request merged**